src/app/components/shell/microsurvey-popup/adoption-content.js

```diff
@@ -1,21 +1,65 @@
 import React, {useRef} from 'react';
 import useUserContext from '~/contexts/user';
-import useAdoptions from '~/models/renewals';
+import {useLocation} from 'react-router-dom';
+import {useToggle} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import cookie from '~/helpers/cookie';
+
+const DISMISSED_KEY = 'renewal_dialog_dismissed';
+const YESTERDAY = Date.now() - 60 * 60 * 24 * 1000;
+
+function useCookieKey(key) {
+    return React.useReducer(
+        (_, value) => {
+            cookie.setKey(key, value);
+            return value ? value : '0';
+        },
+        cookie.hash[key]
+    );
+}
 
 export default function useAdoptionMicrosurveyContent() {
     const {userModel} = useUserContext();
-    const {first_name: name, uuid} = userModel || {};
-    const adoptions = useAdoptions(uuid);
-    const ready = adoptions.Books?.length > 0;
+    const {first_name: name} = userModel || {};
+    const [cookieValue, setCookieValue] = useCookieKey(DISMISSED_KEY);
+    const recentlyDismissed = React.useMemo(
+        () => +cookieValue > YESTERDAY,
```

Comment:
I prefer conversion to number by using the number constructor so it's clearer what's occurring: `Number(cookieValue) > YESTERDAY`